pkg/proxy/iptables/proxier.go

```diff
@@ -1356,6 +1359,11 @@ func writeLine(buf *bytes.Buffer, words ...string) {
 	}
 }
 
+func writeBytes(buf *bytes.Buffer, bytes []byte) {
```

Comment:
this function is repeated again move to pkg/util    DRY

Reply:
Re moving to pkg/util - I'm not a fan of large pkg/util, because when you need something you will not find it anyway in such case. This function is so extremely simple that because of lack of obvious place to put it, I'm leaving it here.